src/repository.c

```diff
@@ -62,7 +62,8 @@ static const struct {
 	{ GIT_REPOSITORY_ITEM_COMMONDIR, GIT_REPOSITORY_ITEM_GITDIR, "worktrees", true }
 };
 
-static int check_repositoryformatversion(git_config *config);
+static int check_repositoryformatversion(git_config *config, int *version);
```

Comment:
Style nit: we specify "out" type params first.  eg, this prototype should be:

```c
static int check_repositoryformatversion(int *version, git_config *config);
```

Reply:
Fixed.